integration-tests/contracts/contract_loader.go

```diff
@@ -368,74 +362,44 @@ func (e *EthereumContractLoader) LoadWERC20Mock(addr common.Address) (WERC20Mock
 	}, err
 }
 
-func (e *EthereumContractLoader) LoadVRFCoordinatorV2_5(addr string) (VRFCoordinatorV2_5, error) {
+func LoadVRFCoordinatorV2_5(seth *seth.Client, addr string) (VRFCoordinatorV2_5, error) {
 	address := common.HexToAddress(addr)
-	instance, err := e.client.LoadContract("VRFCoordinatorV2_5", address, func(
-		address common.Address,
-		backend bind.ContractBackend,
-	) (interface{}, error) {
-		return vrf_coordinator_v2_5.NewVRFCoordinatorV25(address, backend)
-	})
+	abi, err := vrf_coordinator_v2_5.VRFCoordinatorV25MetaData.GetAbi()
 	if err != nil {
-		return nil, err
+		return &EthereumVRFCoordinatorV2_5{}, fmt.Errorf("failed to get VRFCoordinatorV2_5 ABI: %w", err)
 	}
-	return &EthereumVRFCoordinatorV2_5{
-		address:     &address,
-		client:      e.client,
-		coordinator: instance.(*vrf_coordinator_v2_5.VRFCoordinatorV25),
-	}, err
-}
+	seth.ContractStore.AddABI("VRFCoordinatorV2_5", *abi)
+	seth.ContractStore.AddBIN("VRFCoordinatorV2_5", common.FromHex(vrf_coordinator_v2_5.VRFCoordinatorV25MetaData.Bin))
 
-func (e *EthereumContractLoader) LoadVRFv2PlusLoadTestConsumer(addr string) (VRFv2PlusLoadTestConsumer, error) {
-	address := common.HexToAddress(addr)
-	instance, err := e.client.LoadContract("VRFV2PlusLoadTestWithMetrics", address, func(
-		address common.Address,
-		backend bind.ContractBackend,
-	) (interface{}, error) {
-		return vrf_v2plus_load_test_with_metrics.NewVRFV2PlusLoadTestWithMetrics(address, backend)
-	})
+	contract, err := vrf_coordinator_v2_5.NewVRFCoordinatorV25(address, seth.Client)
```

Comment:
⚠️ **very important** ⚠️ 
this has to be changed everywhere, where we create a new Geth wrapper, either when loading or deploying:

Use:
```
wrappers.MustNewWrappedContractBackend(nil, client)
```
instead of:
```
seth.Client
```

So that this
```
contract, err := vrf_coordinator_v2_5.NewVRFCoordinatorV25(address, seth.Client)
```
becomes:
```
contract, err := vrf_coordinator_v2_5.NewVRFCoordinatorV25(address, wrappers.MustNewWrappedContractBackend(nil, client))
```

otherwise `eth_call` **will always fail** for older versions of Geth node.

Reply:
@Tofel ok, let me change it

Reply:
@Tofel fixed in 3703d86149ebf8a38496a7538ed9a0e68518e3ab